src/utils/time.ts

```diff
@@ -35,6 +35,32 @@ export function formatDateFromBatchId(batchId: number): string {
   return formatDistanceToNow(date, { addSuffix: true })
 }
 
+/**
+ * Calculates seconds remaining in current batch
+ * Assumes local time is accurate and can be used as source of truth
+ */
+export function getSecondsRemainingInBatch(): number {
+  return BATCH_TIME - (getEpoch() % BATCH_TIME)
+}
+
+export function formatSecondsToMinutes(seconds: number): string {
```

Comment:
I would call this just `formatSeconds` since it can be missleasing to think that `30` --> `30s` and not `0.5m`

Reply:
See https://github.com/gnosis/dex-react/pull/542